frontend/src/utils/ropeApi.ts

```diff
@@ -283,7 +283,7 @@ export const ropeApi = {
       course_shortname: string
       creator_email: string
       status: string
-      course_id: string
+      course_id: number
```

Comment:
If we decide to update this to `number` we should update `courses.test.tsx ` to reflect this change in the mocks and anywhere else it may be relevant.